src/pgroonga-primary-maintainer.c

```diff
@@ -209,6 +213,83 @@ pgroonga_primary_maintainer_wal_size_check_all(void)
 	pgstat_report_activity(STATE_IDLE, NULL);
 }
 
+static bool
+checkPGroongaPrimaryMaintainerHours(char **newval,
+									void **extra,
+									GucSource source)
+{
+	char *rawstring;
+	List *elemlist;
+	ListCell *l;
+	int i = 0;
+	bool hours[HOURS_SIZE] = {};
+
+	for (i = 0; i < HOURS_SIZE; i++)
+	{
+		hours[i] = false;
+	}
+
+	if (*newval == NULL || *newval[0] == '\0')
+		return true;
+
+	rawstring = pstrdup(*newval);
+	if (!SplitDirectoriesString(rawstring, ',', &elemlist))
+	{
+		list_free_deep(elemlist);
+		pfree(rawstring);
+		ereport(LOG,
+				(errcode(ERRCODE_SYNTAX_ERROR),
+				 errmsg("invalid syntax in pgroonga_primary_maintainer.hour")));
```

Comment:
Is this correct?
It seems that we need to call `GUC_check_err*()` instead:
* https://github.com/postgres/postgres/blob/c61c0cb3a99285db36806b592bd8c540c98757e5/src/include/utils/guc.h#L443-L453
* https://github.com/postgres/postgres/blob/c61c0cb3a99285db36806b592bd8c540c98757e5/src/backend/replication/syncrep.c#L1009-L1014